Dockerfile

```diff
@@ -8,7 +17,7 @@
 # versions to be built while allowing the new versions to make changes
 # that are not backwards compatible.
 #
-FROM quay.io/cilium/cilium-builder:2018-10-29 as builder
+FROM cilium/cilium-builder:jarno-test as builder
```

Comment:
Change image tag

Reply:
Built the new builder image in quay.io, with tag 2018-11-01. It is now only 235MB ;-)